codelists/actions.py

```diff
@@ -806,10 +806,12 @@ def clone_codelist(codelist, new_owner):
         while True:
             if n > 10:
                 raise ValueError("No more than 10 clones of a codelist can be created.")
```

Comment:
question: Unrelated to this PR, but I was looking at this code. I haven't run it, and I may be being very silly, but is there an off-by-one error here?

Let's assume all the possible clones already exist and we loop until reaching this exception.

1. n starts at 0.
2. The first clone is `clone` with no numeric suffix.
3. n increments by 1 if the new name or slug exists.
4. We get `clone1` … `clone10` because when n is 10 we don't raise an exception (we stop when n is 11).
5. `clone` plus `clone1`…`clone10` is 11, by my count.